app/controllers/votes_controller.rb

```diff
@@ -0,0 +1,19 @@
+class VotesController < ApplicationController
+
+  def create
+    if params[:vote_type] == "upvote"
+      parent.votes.create(value: 1)
```

Comment:
how could we extract this code out? what if we could call `upvote` or `downvote` on a parent rather than specifying a value?